libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
@michaelklishin this seems to support the documentation: https://gist.github.com/majormoses/da89381260a814046e4003d677c110e1